Chat_App/Chat_App/UI/BubbleView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  ChatBubbleView.swift
+//  Chat_App
+//
+//  Created by Gi Oo on 01.05.23.
+//
+
+import UIKit
+
+class BubbleView: UIView {
+
+  var senderBubbleColor = UIColor(red: 218, green: 195, blue: 255, alpha: 1)
```

Comment:
I suggest making this property private and declaring it using the let keyword. This would limit its access and scope